content/docs/nav.yml

```
@@ -133,19 +133,19 @@
       title: طريقة إجراء الإختبارات
     - id: testing-environments
       title: بيئة ومحيط الإختبارات
-- title: Concurrent Mode (Experimental)
```

Doesn't anyone think that `الوضع المتزامن` is shaded for newcomers !!

```suggestion
- title: الوضع المتزامن - Concurrent Mode (تجريبي)
```

---

@3imed-jaberi how about `وضعية التزامن`

---

Thank you @asantarissy, let's keep this label until the next nav update (exit from the expremental).
You know that this update will drag a lot of changes behind it.
It may be useless.